handlerconfigurer.go

```diff
@@ -0,0 +1,191 @@
+package configkit
+
+import (
+	"strings"
+
+	"github.com/dogmatiq/configkit/message"
+	"github.com/dogmatiq/dogma"
+)
+
+// handlerConfigurer is an implementation of the configurer interfaces for
```

Comment:
can we say this:
```
handlerConfigurer is an implementation of dogma.*Configurer interfaces for
```

Reply:
There's a `dogma.AggregateConfigurer` that this *doesn't* implement so I'd rather list them explicitly.